### ISSUE TITLE
Enqueue page to UploadWorker when the user tries to upload it while offline

### ISSUE THREAD

**Issue by malinajirka** [wordpress-mobile/WordPress-Android#11129]
When the user tries to upload a post in offline, we enqueue the post to UploadWorker which is run periodically when certain conditions are met. Make sure the same behavior works for pages.

Note: I think it might start working automatically when #11126 is implemented. The logic for posts is [here](https://github.com/wordpress-mobile/WordPress-Android/blob/76df20a836872dc5ed186ec4d5f98d679368368b/WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadUtils.java#L169) and if I'm not wrong, the same code is executed for pages.

**Comment by jd-alexander** [comment]
You were right. I ran a test and the page was enqueued. I also utilized the debugger to ensure that this [line](https://github.com/wordpress-mobile/WordPress-Android/blob/76df20a836872dc5ed186ec4d5f98d679368368b/WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadUtils.java#L169) was hit. 

These were the steps I followed. 
1. Opened the app while it was in airplane mode. 
2. On the `My Site` tab I clicked the create FAB, selected `Site Page`.
3. I created a new page and clicked publish. 
4. Once I re-enabled the internet, the page was uploaded.
